apps/api/src/api/controllers/session.controller.ts

```diff
@@ -0,0 +1,94 @@
+import { GetWidgetUrlLocked, GetWidgetUrlRefresh, GetWidgetUrlResponse, RampDirection } from "@packages/shared";
+import { NextFunction, Request, Response } from "express";
+import httpStatus from "http-status";
+import { APIError } from "../errors/api-error";
+import quoteService from "../services/ramp/quote.service";
+
+const BASE_WIDGET_URL = process.env.RAMP_WIDGET_URL || "https://app.vortexfinance.co/widget";
```

Comment:
Nit: Both work but going forward we should share 
```suggestion
const BASE_WIDGET_URL = process.env.RAMP_WIDGET_URL || "https://www.vortexfinance.co/widget";
```